test/e2e/util/e2e.ts

```diff
@@ -0,0 +1,107 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import axios, { AxiosInstance } from 'axios'
+import { Agent } from 'https'
+import { stringify } from 'querystring'
+
+import { CheHelper } from '../../../src/api/che'
+import { KubeHelper } from '../../../src/api/kube'
+import { OpenShiftHelper } from '../../../src/api/openshift'
+
+//Utilities to help e2e tests
+export class E2eHelper {
+  protected kubeHelper: KubeHelper
+  protected che: CheHelper
+  protected oc: OpenShiftHelper
+
+  private readonly axios: AxiosInstance
+  constructor() {
+    this.kubeHelper = new KubeHelper({})
+    this.che = new CheHelper({})
+    this.oc = new OpenShiftHelper()
+    const httpsAgent = new Agent({ rejectUnauthorized: false })
```

Comment:
What is the reason to have `rejectUnauthorized` turned off?

Reply:
Is just for test purpose.